gap/FinSetsForCAP.gi

```diff
@@ -623,6 +627,12 @@ AddIsMonomorphism( FinSets,
     
 end );
 
+##
+AddIsSplitMonomorphism( FinSets,
+  function ( phi )
+    return IsInitial( Range( phi ) ) or ( not IsInitial( Source( phi ) ) and IsMonomorphism( phi ) );
```

Comment:
`IsInitial` is derived from `UniversalMorphismFromInitialObject` and `IsIsomorphism`. `IsIsomorphism` in turn is derived from `IsMonomorphism` and `IsSplitEpimorphism` but could also be derived from `IsSplitMonomorphism` and `IsEpimorphism`. To make sure we do not run into infinite recursions I propose to implement a primitive version of `IsInitial`.

Reply:
This is included in the current version.